cloud/dump_http.go

```diff
@@ -0,0 +1,14 @@
+package cloud
+
+import (
+	"net/http"
+	"net/http/httputil"
+)
+
+func dumpRequest(req *http.Request) ([]byte, error) {
```

Comment:
*nitpick* I'd add some description because it's not clear what it does with regards the func name but otherwise it's good

Reply:
@sixstone-qq done.

I also removed the `dump_http.go` file and moved th function to `cloud/cloud.go`)